docs/data/migration/migration-data-grid-v6/migration-data-grid-v6.md

```diff
@@ -108,6 +108,47 @@ The `legacy` bundle that used to support old browsers like IE 11 is no longer i
 If you need support for IE 11, you will need to keep using the latest version of the `v6` release.
 :::
 
+### Drop webpack 4 support
```

Comment:
However, I'm not sure that this is the most appropriate heading for this section. 🙈 
Technically, we only made our bundles slightly more modern, the problems with Webpack can be overcome by using transformers. 🤔 
```suggestion
### Drop Webpack 4 support
```

Reply:
I think it's OK.
Alternatively, I can call it "Drop out of the box Webpack 4 support", but it's too verbose 😅